packages/app-headless-cms/src/admin/config/contentEntries/editor/ContentEntryEditorConfig.tsx

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from "react";
+import { createConfigurableComponent } from "@webiny/react-properties";
+import { Actions, ActionsConfig } from "./Actions";
+
+const base = createConfigurableComponent<ContentEntryEditorConfig>("ContentEntryEditorConfig");
+
+export const ContentEntryEditorConfig = Object.assign(base.Config, { Actions });
+
+export const ContentEntryEditorWithConfig = base.WithConfig;
+
+interface ContentEntryEditorConfig {
+    actions: ActionsConfig;
+}
+
+export function useContentEntryEditorConfig() {
+    const config = base.useConfig();
+
+    const actions = config.actions || {};
```

Comment:
Is the actions an object or an array? Check line 22 in this file.
You will see the error when you set that the `actions` property on line `12` is possibly `undefined`.